extensions/notebook/package.nls.json

```diff
@@ -41,5 +41,6 @@
 	"title.closeJupyterNotebook": "Close Jupyter Notebook",
 	"title.revealInBooksViewlet": "Reveal in Books",
 	"title.createJupyterBook": "Create Book (Preview)",
-	"title.openNotebookFolder": "Open Notebooks in Folder"
+	"title.openNotebookFolder": "Open Notebooks in Folder",
+	"title.openRemoteJupyterBook": "Add Remote Jupyter Book"
```

Comment:
>Add Remote Jupyter Book [](start = 33, length = 23)

Did you check this with PM? 'Open External Jupyter Book' ? Just confirm it with PM Alan before getting it in.

Reply:
Yeah, actually we had a meeting with Raden and decided that Add describes better the experience, since we are adding a remote book into the Notebooks Viewlet.